app/controllers/application_controller.rb

```diff
@@ -11,7 +11,8 @@ class ApplicationController < ActionController::Base
   include Vmdb::Logging
 
   if Vmdb::Application.config.action_controller.allow_forgery_protection
-    protect_from_forgery :secret =>  MiqDatabase.connected? ? MiqDatabase.first.csrf_secret_token : SecureRandom.hex(64), :except => :csp_report, :with => :exception
+    db = MiqDatabase.connected? ? MiqDatabase.first : nil
```

Comment:
In that case wouldn't I get an exception in the next line if the database is not connected? I would think in that case db wouldn't be defined right?